interfaces/inc/c_pal/threadpool.h

```diff
@@ -21,10 +21,12 @@ extern "C" {
 #endif
 
 typedef struct THREADPOOL_TAG THREADPOOL;
-typedef struct TIMER_INSTANCE_TAG* TIMER_INSTANCE_HANDLE;
+typedef struct TIMER_TAG TIMER;
 typedef struct THREADPOOL_WORK_ITEM_TAG THREADPOOL_WORK_ITEM;
 typedef void (*THREADPOOL_WORK_FUNCTION)(void* context);
 
+THANDLE_TYPE_DECLARE(TIMER);
```

Comment:
>TIMER [](http://example.com/codeflow?start=21&length=5)

I think this should be named THREADPOOL_TIMER. To be in sync with THREADPOOL_WORK_FUNCTION

Reply:
done